sandbox/errorHandling.lua

```diff
@@ -169,6 +162,10 @@ function errorHandling:_Artifact()
 	local create = Artifact
 	return function()
 		local artifact = create()
+		artifact.onCollision = self:_autoWrapArgX(artifact.onCollision,2)
+		artifact.onPlayerCollision = self:_autoWrapArgX(artifact.onPlayerCollision,2)
+		artifact.onPickUp = self:_autoWrapArgX(artifact.onPickUp,2)
+		artifact.onPickup = self:_autoWrapArgX(artifact.onPickup,2)

```

Comment:
Are there two callbacks on this, one with a lower case u and another with an upper case U? Your title says asteroid error handling and I was looking for asteroid functions, but then I figured out you meant artifact